frontend/src/patientApp/timeOfTest/TestResult.tsx

```diff
@@ -0,0 +1,134 @@
+import { connect, useSelector } from "react-redux";
+
+import { formatFullName } from "../../app/utils/user";
+import { RootState } from "../../app/store";
+import { Patient } from "../../app/patients/ManagePatients";
+
+const serializeResult = (result: string) => {
+  switch (result) {
+    case "POSITIVE":
+      return "Positive";
+    case "NEGATIVE":
+      return "Negative";
+    default:
+      return "Inconclusive";
+  }
+};
+
+const TestResult = () => {
+  const patient = useSelector<RootState, Patient>((state) => state.patient);
+  const fullName = formatFullName(patient as any);
+  const dateTested = new Date(patient.lastTest.dateTested).toLocaleDateString();
+  const deviceType = patient.lastTest.deviceType;
+  const result = serializeResult(patient.lastTest.result);
+
+  return (
+    <main className="patient-app padding-top-105 padding-bottom-4 bg-base-lightest">
+      <div className="grid-container maxw-tablet">
+        <div className="card usa-prose">
+          <h1 className="font-heading-lg">SARS-CoV-2 result</h1>
+          <h2 className="font-heading-sm">Patient</h2>
+          <p className="margin-top-05">{fullName}</p>
+          <div className="grid-row">
+            <div className="grid-col usa-prose">
+              <h2 className="font-heading-sm">Test result</h2>
+              <p className="margin-top-05">{result}</p>
+            </div>
+            <div className="grid-col usa-prose">
+              <h2 className="font-heading-sm">Test date</h2>
+              <p className="margin-top-05">{dateTested}</p>
+            </div>
+          </div>
+          <h2 className="font-heading-sm">Test device</h2>
+          <p className="margin-top-05">{deviceType}</p>
+          <h2 className="font-heading-sm">Notes</h2>
+          <TestResultNotes result={result} />
+          <p>
+            For further guidance, please consult the{" "}
+            <a href="https://www.cdc.gov/coronavirus/2019-ncov/if-you-are-sick/end-home-isolation.html">
+              Centers for Disease Control and Prevention (CDC)
+            </a>{" "}
+            website or contact your local health department.
+          </p>
+        </div>
+      </div>
+    </main>
+  );
+};
+
+interface TestResultNotesProps {
+  result: string;
```

Comment:
I would definitely recommend an enum type here; definitely raised an eyebrow because I know that the result was _previously_ uppercase but this is now referring to the serialized version. Would recommend better types for readability and to prevent mistakes.